### PR TITLE
Feature/33705 upload version image

### PR DESCRIPTION
**Pull request by heiniovason**
Now possible to upload file in django admin site for **ImageVersion**.

Note that there currently is input-validation, and no progress bar while uploading file; have patience padowans.

### CODE REVIEW

File: admin_site/system/views.py [hunk 1]
```diff
@@ -1874,3 +1895,50 @@ def get_context_data(self, **kwargs):
                 pc[pn] = pv
             pcs.append(pc)
         return pcs
+
+
+class ImageVersionsView(SiteMixin, SuperAdminOrThisSiteMixin, ListView):
+    """Image Versions are are sorted by major versions in the view
```
Comment (magenta-af) [inline admin_site/system/views.py]: This message has a typo (`are are`) and is not terribly explanatory. How about this?

```suggestion
    """Displays all of the image versions that this site has access to (i.e.,
    all versions released before the site's last_version datestamp).
```

File: admin_site/system/views.py [hunk 2]
```diff
@@ -1874,3 +1895,50 @@ def get_context_data(self, **kwargs):
...
+        for minor_version in versions:
+            major_versions_set.add(minor_version.img_vers[:1])
+        major_versions_list = list(major_versions_set)
+        major_versions_list.sort(reverse=True)
+
+        context["major_versions"] = major_versions_list
+
+        url_ref_vers = self.kwargs.get(
+            'major_version',
+            major_versions_list[0]
```
Comment (magenta-af) [inline admin_site/system/views.py]: I remark that this line makes the administration system crash if no versions are configured:

Screenshot_2020-02-14 IndexError at site magenta image-versions

File: admin_site/system/views.py [hunk 3]
```diff
@@ -1874,3 +1895,50 @@ def get_context_data(self, **kwargs):
...
+
+        site_uid = self.kwargs.get('site_uid')
+        site_obj = Site.objects.get(uid=site_uid)
+        site_allowed = False
+        if site_obj.last_version:
+            site_allowed = True
+
+        context["site_allowed"] = site_allowed
+
+        versions = ImageVersion.objects.all()
```
Comment (magenta-af) [inline admin_site/system/views.py]: Er... is this right? Surely we should only show images whose release date is less than or equal to the customer's `last_version` date.

File: admin_site/system/views.py (hunk 1 above)
Comment (heiniovason) [reply admin_site/system/views.py]: Thank you

File: admin_site/system/views.py (hunk 2 above)
Comment (heiniovason) [reply admin_site/system/views.py]: Check

**Pull request merged**